### PR TITLE
Version card/version history redesign

### PR DESCRIPTION
**Pull request by jgruica**
Screen Shot 2020-10-01 at 3 50 21 PM
Screen Shot 2020-10-01 at 3 48 58 PM
Screen Shot 2020-10-01 at 3 49 45 PM
Screen Shot 2020-10-02 at 8 55 44 AM
Screen Shot 2020-10-02 at 8 35 15 AM

### CODE REVIEW

File: kotsadm/web/src/components/modals/UpdateCheckerModal.jsx [hunk 1]
```diff
@@ -128,7 +128,7 @@ export default class UpdateCheckerModal extends React.Component {
         className="Modal SmallSize"
       >
         <div className="u-position--relative flex-column u-padding--20">
-          <span className="u-fontSize--largest u-fontWeight--bold u-color--tuna u-marginBottom--15">Configure automatic updates</span>
+          <span className="u-fontSize--largest u-fontWeight--bold u-color--tuna u-marginBottom--15">Automatic updates checks</span>
```
Comment (GraysonNull) [inline kotsadm/web/src/components/modals/UpdateCheckerModal.jsx:131]: I think this should say "Configure automatic update checks"

File: kotsadm/web/src/components/apps/DashboardCard.jsx [hunk 2]
```diff
@@ -168,36 +262,39 @@ export default class DashboardCard extends React.Component {
...
             : showOnlineUI ?
-              <div className="flex alignItems--center">
-                <button className="btn primary blue u-marginTop--10" onClick={isUpdateAvailable ? redirectToDiff : onCheckForUpdates}>{isUpdateAvailable ? "Show Update" : "Check for update"}</button>
-                <span className="icon settings-small-icon u-marginLeft--5 u-cursor--pointer u-marginTop--10" onClick={this.props.showUpdateCheckerModal} data-tip="Configure automatic updates"></span>
-                <ReactTooltip effect="solid" className="replicated-tooltip" />
+              <div className="flex1 flex-column" style={{ flexGrow: 1 }}>
+                {this.renderVersionAvailable(downstream)}
+                <div className="flex alignItems--center">
+                  <button className="btn primary blue u-marginTop--10" onClick={isUpdateAvailable ? redirectToDiff : onCheckForUpdates}>Check for update</button>
+                  <span className="icon settings-small-icon u-marginLeft--5 u-cursor--pointer u-marginTop--10" onClick={this.props.showUpdateCheckerModal} data-tip="Configure automatic updates"></span>
```
Comment (GraysonNull) [inline kotsadm/web/src/components/apps/DashboardCard.jsx:290]: Let's change this tooltip to be "Configure automatic update checks"

File: kotsadm/web/src/components/modals/UpdateCheckerModal.jsx (hunk 1 above)
Comment (markpundsack) [reply kotsadm/web/src/components/modals/UpdateCheckerModal.jsx:131]: What about something smaller, like "Configure checks"?

Comment (markpundsack) [reply kotsadm/web/src/components/modals/UpdateCheckerModal.jsx:131]: If it's only on hover, then we can get away with more verbose description, if that's clearer.

Comment (markpundsack) [reply kotsadm/web/src/components/modals/UpdateCheckerModal.jsx:131]: Oh wait, this is in the modal title itself. Yeah, let's go with verbose.

**Comment by markpundsack** [comment]
In the last image, how come the all versions row for the currently deploying version doesn't have a status icon? The app banner does. 

Likewise, how come the older version doesn't show any status at all?

@GraysonNull Thinking about the deploying icon, it looks like a grayed out do-not-enter sign. Can you come up with a better icon indicating ongoing deployment?

**Comment by GraysonNull** [comment]
Yeah agreed, it's not a great icon for that. How about just showing the loading spinner in place of an icon? I think that would make sense to show that there plus would give some good feedback that something is actually happening.

**Comment by markpundsack** [comment]
Oh interesting, you mean an animated spinner. That's kinda brilliant.

**Comment by GraysonNull** [comment]
Yeah exactly, seems kinda like a perfect use for it.

**Comment by jgruica** [comment]
@GraysonNull updated code and screenshots with deploying spinner

**Review by GraysonNull — APPROVED** [review]

**Review by markpundsack — APPROVED** [review]

**Pull request merged**